### ISSUE TITLE
Actions for 'Entry delay started' do not happen

### ISSUE THREAD

**Issue by ichilton** [nielsfaber/alarmo#678]
### Alarmo Version

v1.9.7

### HA Version

2023.3.1

### Bug description

I want to make a buzzer beep when someone enters and triggers an entry zone.

I have an action as follows:

Event: Entry delay started.
Area: House
Entity: buzzer.
Turn on.

...but it doesn't seem to work.

Actions on exit delay or triggered work fine.

**Comment by nielsfaber** [comment]
I created a notification for this:

<img src="https://user-images.githubusercontent.com/16861605/226158205-04ccb6f3-f51c-4aa9-93e6-be8e47ad3ac7.png" width="75%" height="75%">

As soon as I trigger a sensor the alarm goes to pending state:
<img src="https://user-images.githubusercontent.com/16861605/226158246-8cce41bd-af38-43f3-801e-01d844e09486.png" width="75%" height="75%">

And the notification is triggered:
<img src="https://user-images.githubusercontent.com/16861605/226158289-753d76ac-fdaf-430a-9450-6d55e69095e1.png" width="75%" height="75%">

In summary, it works fine for me.
Please provide more info on how to reproduce the problem, otherwise I can't help you.

**Comment by ichilton** [comment]
So... I think I found out what's happening here. I added a chime action to my door contacts and I think that's then turning the buzzer off straight after the entry buzz has started.

Will remove it and re-test.

Thanks,

Ian
